### DISCUSSION TITLE
Account-wide Last.fm scrobbling

### DISCUSSION THREAD

**Discussion by PedroHLC** [tidal-music/discussions#51] [Ideas & Feature requests]
### What?

Instead of per-browser (TIDAL Web) and third-party-apps (Android), scrobble to Last.fm account-wide. Not necessarily server-side, but authorize it once and apply to all future playback from any device. 

### Why?

- This would expand support for scrobbling to TVs, Wearables, and other TIDAL Connect devices that don't and/or won't ever support Last.fm;
- This would make it necessary to connect to Last.fm only once per user;
- Current implementation easily fails to scrobble on Android-devices because of those poorly coded vendor-based battery-optimization daemons killing the 3rd-party scrobblers' apps;
- Spotify made this exact move years ago, instantly supporting edge-case systems like Samsung Tizen TVs and Chromecast -- this would help users migrating from Spotify to TIDAL;
- Current implementation on TIDAL Web is misleading, since having the connection button within the “Account tab” makes one correlate it to being account-wide.
- There are numerous users facing issues with the current approach:
  - https://www.reddit.com/r/TIdaL/comments/12w9665/inconsistant_scrobbling_to_lastfm/ (contains Account-wide suggestion)
  - https://www.reddit.com/r/TIdaL/comments/13mp99d/tidal_and_lastfm_issues/
  - https://www.reddit.com/r/TIdaL/comments/17zyxjk/tidal_and_lastfm_scrobbler/
  - https://www.reddit.com/r/lastfm/comments/126vam4/tidal_scrobbles_issue/
  - https://www.reddit.com/r/TIdaL/comments/124fi95/lastfm_scrobbling_doesnt_work_on_ios/
  - https://www.reddit.com/r/TIdaL/comments/sqxqg8/cant_connect_tidal_to_lastfm/
  - https://www.reddit.com/r/TIdaL/comments/168ph7z/for_those_who_may_have_given_it_up_lastfm_now/ (contains users who left TIDAL because of Last.fm)
  - https://www.reddit.com/r/TIdaL/comments/18g1m59/whats_the_best_lastfm_scrobbler_for_android/
  - https://www.reddit.com/r/TIdaL/comments/xa7n1h/how_do_i_connect_tidal_to_lastfm_on_android/
  - https://www.reddit.com/r/lastfm/comments/13iarvm/alternative_lastfm_scrobbler_on_ios_for_tidal/

**Comment by stefanmoro** [comment]
As this is a feature request for our 1st party apps, please reach out to TIDAL support with the request. Please see [this message](https://github.com/orgs/tidal-music/discussions/1) for information about TIDAL Developer and this forum.

Thanks.
